build.gradle.kts

```diff
@@ -44,7 +44,7 @@ subprojects {
 
     apply(plugin = rootProject.libs.plugins.jvm.get().pluginId)
 
-    testDb("h2") {
+    testDb("h2_v2") {
```

Comment:
Changing this string means it would also need to change for the inclusion of Spring tests.
The number of tests run for this branch dropped because all Spring tests are being skipped now, even by H2 (confirmed locally and on [TC build](https://exposed.teamcity.com/buildConfiguration/Exposed_Build/8755?hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandBuildDeploymentsSection=false&showLog=8755_233514_593&logView=flowAware)).
This property in `TestDbDsl.kt` needs to be changed to also match in whatever way you think is good:
```kt
internal val ignoresSpringTests = name != "h2_v2"
```

Another location I found it being used is in `H2Tests.kt/testH2VersionIsCorrect()`

Reply:
Please also check how the dependency of the correct version of H2 driver is provided

Reply:
As I can see the test `testH2VersionIsCorrect()` checks that. But the problem with the test was that it was able to skip all the check in the case of wrong system property `"exposed.test.name"`, so I updated the test that it fails if that property has unexpected value (simply added assert `assertTrue(systemTestName == "h2_v2" || systemTestName == "h2_v1")`)